src/components/Proposal/View/ProposalProcess.tsx

```diff
@@ -2,60 +2,58 @@ import React from 'react'
 
 import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
 
-import Divider from '../../Common/Divider'
-import NewBadge from '../NewBadge/NewBadge'
-
 import './ProposalProcess.css'
+import Section from './Section'
+
+export enum ProcessStatus {
```

Comment:
Shouldn't this be defined in `types.ts` ?

Reply:
This is internal enum for for this component, and shared with the other component using ProposalProcess.